docs/UserGuide.md

```diff
@@ -111,44 +111,44 @@ Example: `view 34`
 | <...> | Indicates the argument that the user should provide in that area |
 
 #### 2.3.2 Add/Edit phone numbers: phone
-Format: `phone <NUMBER 1> /<LABEL>; <NUMBER 2> /<LABEL>...`
+Format: `p/<NUMBER 1> l/<LABEL> p/<NUMBER 2> l/<LABEL>...`
```

Comment:
Can we mention somewhere that all the phone numbers don't need to be one after the other? Perhaps we can add as aggregated example command like `add n/Alex p/30303030 l/personal e/alex@example.com p/98989898 c/Tesla j/CEO t/neighbour` towards the end in the **Commands Summary**.